docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,242 @@ grand_parent: "General"
+
+Feel free to play with it [here](https://demo.bpmn.io/s/start).
+
+![image](/assets/images/bpmn-start.gif)
+
+There are two ways to include bpmn-js in your application:
+
+* An [all-in-one pre-packaged version of the library](https://unpkg.com/browse/bpmn-js/dist/) allowing you to quickly add BPMN to any web application.
+* The [npm](http://npmjs.com/package/bpmn-js) version, which is more complicated to set up but gives you access to individual library components and allows for easier extensibility.
+
+## Including the Root Element
+
+Both as a viewer or modeler, bpmn-js requires a root DOM element (a simple DIV) on which the rendering is performed. Simply, create a DIV element in the HTML file:
+
+```html
+<!-- BPMN diagram container -->
+<div id="canvas"></div>
+```
+
+## Including the Stylesheet
+
+We should include [diagram-js](/general/getting-started/diagram-js) CSS and [BPMN Icon Font](https://github.com/bpmn-io/bpmn-font) in order to ensure that everything looks good. There're several ways we can achieve that:
+
+We can include the CSS via `link` tag in our HTML file. Note that these CSS files are shipped with with the bpmn-js distribution under [dist/assets folder](https://unpkg.com/browse/bpmn-js/dist/assets/).
+
+```html
+<link rel="stylesheet" href="bpmn-js/dist/assets/diagram-js.css" />
+<link rel="stylesheet" href="bpmn-js/dist/assets/bpmn-font/css/bpmn.css" />
+```
+
+If we've install the bpmn-js via [NPM](#including-via-npm), we can additionally include these CSS files using `import` statement in our Javascript file:
+
+```javascript
+import "bpmn-js/dist/assets/diagram-js.css";
+import "bpmn-js/dist/assets/bpmn-font/css/bpmn-embedded.css";
+```
+
+## Using bpmn-js as Viewer
+
+bpmn-js may be used as a viewer. In that case the user would not be able to interact with the diagram to modify the existing BPMN 2.0 XML, but would only be able to view it. In addition to viewer, bpmn-js offers a navigated viewer which enables [move-keyboard](/modules/common-modules/keyboard-move), [move-canvas](/modules/common-modules/move-canvas) and [zoom-scroll](/modules/common-modules/zoom-scroll) [modules](/modules/adding-customizing-removing) by default.
+
+### Including the Pre-packaged Version
+
+For pre-packaged version, we should include the bpmn-js viewer via HTML script tag:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-viewer.production.min.js"></script>
+```
+
+or for the navigated viewer:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-navigated-viewer.production.min.js"></script>
+```
+
+### Including via NPM
+
+First, install bpmn-js via NPM:
+```
+npm install bpmn-js
+```
+
+Then import the BPMN viewer via an ES:
```

Comment:
What is "an ES"? Do you mean "an ES module"?

Reply:
Ah yeah, 🤦 Done 👍